app/src/main/java/protect/card_locker/DBHelper.java

```diff
@@ -175,5 +175,29 @@ public int getLoyaltyCardCount()
 
         return numItems;
     }
+
+    public int getFilteredLoyaltyCardCount(String filter)
```

Comment:
Could this call use getLoyaltyCardCursor(), so that the query is not listed twice?

Reply:
Yup, will fix that